### PR TITLE
fence_ipmilan: fix typo in description

### PR DESCRIPTION
**Pull request by razo7**
Add spaces in the description

### CODE REVIEW

**Comment by oalbrigt** [comment]
Nice catch. You also have to update `https://github.com/ClusterLabs/fence-agents/blob/main/agents/ipmilan/fence_ipmilan.py#L206`. Simply add a space before the `\` on that line, commit and push.

File: tests/data/metadata/fence_imm.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <symlink name="fence_ipmilanplus" shortdesc="Fence agent for IPMIv2 lanplus"/>
 <symlink name="fence_imm" shortdesc="Fence agent for IBM Integrated Management Module"/>
 <symlink name="fence_idrac" shortdesc="Fence agent for Dell iDRAC"/>
-<longdesc>fence_ipmilan is an I/O Fencing agentwhich can be used with machines controlled by IPMI.This agent calls support software ipmitool (http://ipmitool.sf.net/). WARNING! This fence agent might report success before the node is powered off. You should use -m/method onoff if your fence device works correctly with that option.</longdesc>
+<longdesc>fence_ipmilan is an I/O Fencing agent which can be used with machines controlled by IPMI. This agent calls support software ipmitool (http://ipmitool.sf.net/). WARNING! This fence agent might report success before the node is powered off. You should use method off follows on if your fence device works correctly with that option.</longdesc>
```
[the remark this answers is not in the document]
Comment (razo7) [reply tests/data/metadata/fence_imm.xml:9]: I wasn't aware of the `-m/method` :)

[the remark this answers is not in the document]
Comment (oalbrigt) [reply tests/data/metadata/fence_imm.xml:9]: It's easy to make mistakes when doing it manually :)

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**